modules/tinymce/src/themes/silver/test/ts/browser/editor/header/PromotionTest.ts

```diff
@@ -0,0 +1,35 @@
+import { UiFinder } from '@ephox/agar';
+import { context, describe, it } from '@ephox/bedrock-client';
+import { SugarElement } from '@ephox/sugar';
+import { TinyHooks } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+
+describe('browser.tinymce.themes.silver.editor.header.PromotionTest', () => {
+  const promotionClass = 'tox-promotion';
+
+  context('TINY-8840: promotion turned off', () => {
```

Comment:
The Jira number should be on the `it` instead of `context`, as that's what it's specific to.

Reply:
I swear I have done this correctly before...